src/index.tsx

```diff
@@ -212,7 +212,11 @@ export class RCPhoneInput extends React.Component<IProps, IState> {
     if (props.value) {
       selectedCountryGuess = this.guessSelectedCountry(props.value)
       if (selectedCountryGuess) {
-        number = props.value
+        if (props.value.startsWith('+' + selectedCountryGuess.dialCode)) {
+          number = props.value.split(selectedCountryGuess.dialCode)[1]
+        } else {
+          number = props.value
+        }
```

Comment:
everywhere you can use `value` instead `props.value`

Reply:
@favorit this is sperate function.